virttest/qemu_devices/qdevices.py

```diff
@@ -9,21 +9,15 @@
 import logging
 import re
 import traceback
+# pylint: disable=E0611
```

Comment:
I don't see the reason for this pylint exception.  On my environment:

```
$ git diff
diff --git a/virttest/qemu_devices/qdevices.py b/virttest/qemu_devices/qdevices.py
index 5b2957459..f368f67e9 100644
--- a/virttest/qemu_devices/qdevices.py
+++ b/virttest/qemu_devices/qdevices.py
@@ -9,7 +9,6 @@ interact with qemu qdev structure.
 import logging
 import re
 import traceback
-# pylint: disable=E0611
 from collections import OrderedDict
 
 from virttest import qemu_monitor

$ inspekt lint --disable W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003 virttest/qemu_devices/qdevices.py
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011,E1003
Pylint enabled : W0611
Syntax check PASS
```

Reply:
ok, that makes sense.